docs/modules/user-guide/partials/proc_devfile-properties.adoc

```diff
@@ -0,0 +1,258 @@
+[id="proc_devfile-properties_{context}"]
+= Devfile properties 
+
+[role="_abstract"]
+Use this page to work with yaml files of devfile properties. See the following table of contents to go to the devfile property you want to work with:
+
+== SchemaVersion
+
```

Comment:
Introductory sentences do seem redundant often, but maybe they would help to understand this guide? Like "This section describes how to xy schema versions of devfiles". The same goes for the other sections too of course.

Reply:
Update: If you decide to merge the tables into one, introductory sentences like the proposed one would probably become unnecessary.

Reply:
Okay, I think, for now, we'll leave each section as having it's own table...in which case I'll add an introductory sentence to the abstract and individual sections like you suggested!